intranet/scss/components/_block-warning.scss

```diff
@@ -0,0 +1,51 @@
+.block-warning {
+  background-color: $yellow;
+  box-shadow: 0 rem(4px) rem(8px) 0 rgba(0, 0, 0, .1);
```

Comment:
move to header-sticky and don't need style  
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: rem(50px);
+  transition: .3s;
+  z-index: 9;